keras/backend/mxnet_backend.py

```diff
@@ -1880,7 +1898,11 @@ def conv2d(x, kernel, strides=(1, 1), border_mode='valid',
     # Returns
         A tensor, result of 2D convolution.
     """
-    raise NotImplementedError
+    shape = kernel.shape
```

Comment:
you need handle dim_ordering. the following code can handle 3d tensor.


def _layout_from(dim_ordering):
    if dim_ordering == 'default':
        dim_ordering = image_dim_ordering()
    if dim_ordering == 'th':
        return 'NHWC'
    elif dim_ordering == 'tf':
        return 'NCHW'
    else:
        raise ValueError('Unknown dim_ordering ' + str(dim_ordering))